### PR TITLE
Replace Dependency on fieldMap.js with mappings API Call [merge with WWW and PB]

### PR DESCRIPTION
**Pull request by alexweltman**
* goes with PRs in Protobuffers and www
* uses API route created in my www PR
* queries are not auto-capitalized when saved as query rules

### CODE REVIEW

**Comment by craig-cogdill** [comment]
👍

File: packaging/kibana.spec [hunk 1]
```diff
@@ -56,6 +56,10 @@ if [ -e /usr/local/%{name}-%{kibana_version}-linux-x64/refreshKibanaIndex.sh ];
    rm /usr/local/%{name}-%{kibana_version}-linux-x64/refreshKibanaIndex.sh
 fi
 
+if [ -e /usr/local/%{name}-%{kibana_version}-linux-x64/src/public/netmon_libs/fieldMap.js ]; then
```
Comment (KjellKod) [inline packaging/kibana.spec]: This file does not seem to be present on my box, nor on any QA box I have checked.

Comment (KjellKod) [inline packaging/kibana.spec]: The only location for this file is under `/usr/local/www/probe/skin/js/fieldMap.js`

**Comment by KjellKod** [comment]
👍

**Pull request merged**